Examples/CaseStudies/SwiftUICaseStudies/01-GettingStarted-AlertsAndActionSheets.swift

```diff
@@ -48,8 +48,8 @@ let alertAndSheetReducer = Reducer<
       message: .init("This is an action sheet."),
       buttons: [
         .cancel(),
-        .default(.init("Increment"), send: .incrementButtonTapped),
-        .default(.init("Decrement"), send: .decrementButtonTapped),
+        .default(.init("Increment"), action: .send(.incrementButtonTapped)),
+        .default(.init("Decrement"), action: .send(.decrementButtonTapped)),
```

Comment:
`action` is the argument name that `SwiftUI.Alert.Button` constructors take, so this will already more closely model the upstream API.